src/commands/synthetics/__tests__/cli.test.ts

```diff
@@ -122,7 +122,6 @@ describe('run-test', () => {
           followRedirects: toBoolean(overrideEnv.DATADOG_SYNTHETICS_OVERRIDE_FOLLOW_REDIRECTS),
           headers: toStringObject(overrideEnv.DATADOG_SYNTHETICS_OVERRIDE_HEADERS),
           mobileApplicationVersion: overrideEnv.DATADOG_SYNTHETICS_OVERRIDE_MOBILE_APPLICATION_VERSION,
-          pollingTimeout: DEFAULT_POLLING_TIMEOUT,
```

Comment:
Should we add a case for `DATADOG_SYNTHETICS_BATCH_TIMEOUT` in `overrideEnv` above and here in the root level?

Reply:
Good point, thanks 👍